### ISSUE TITLE
services.mopidy.enable = true; fails

### ISSUE THREAD

**Issue by mt-caret** [NixOS/nixpkgs#36205]
## Issue description

When I try to enable mopidy, I get **exactly** the same error as https://github.com/NixOS/nixpkgs/issues/33932:
```
collision between `/nix/store/3d8nq34016rjz5vfy7w36k77nnrvkchk-gst-python-1.10.4/lib/python2.7/site-packages/gi/overrides/__init__.pyo' and `/nix/store/xir2hwxj8870j311hx8jdi63vcspljs0-python2.7-pygobject-3.24.1/lib/python2.7/site-packages/gi/overrides/__init__.pyo'
```

Is this a known issue?

Full error message:
```
$ sudo nixos-rebuild boot
building Nix...
warning: in configuration file '/etc/nix/nix.conf': unknown setting 'signed-binary-caches'
warning: in configuration file '/etc/nix/nix.conf': unknown setting 'signed-binary-caches'
building the system configuration...
warning: in configuration file '/etc/nix/nix.conf': unknown setting 'signed-binary-caches'
these derivations will be built:
  /nix/store/87vchvqv4misz0qxhh7fmxbwp3dihln0-system-path.drv
  /nix/store/08wzmf5v9387449kks199ar2x1j6jl3g-unit-polkit.service.drv
  /nix/store/2h60dd3whbfrsqv1vaklsn71mg93s4lj-dbus-1.drv
  /nix/store/ddsfxdccab6mv1bvxk25k8pf3pp6i2k9-unit-dbus.service.drv
  /nix/store/9ac0dynaq8cm8n8280wmph204fl87hff-user-units.drv
  /nix/store/flrd0spl6qqkssbvfknlrq40g5gccsjw-unit-systemd-fsck-.service.drv
  /nix/store/z32xpl22v64vhw91pcd3d7iyjl2hp1vb-python-2.7.14-env.drv
  /nix/store/lxmzpva4spmz3l4ny79r3lqcwirjsk1x-unit-mopidy-scan.service.drv
  /nix/store/z583bmd414jflbd33d440zgpkzpg44d2-unit-mopidy.service.drv
  /nix/store/hbyfiv3x1hq4gm7jdcsd77fhjwjmzjjf-system-units.drv
  /nix/store/p3jf9jjf8ws884cs20jk2h6ki2yg6zks-etc.drv
  /nix/store/2x07d2aqs981hvkd148wmssylw6wpqxm-nixos-system-dn-17.09.3109.48a76ca6f57.drv
building '/nix/store/87vchvqv4misz0qxhh7fmxbwp3dihln0-system-path.drv'...
building '/nix/store/z32xpl22v64vhw91pcd3d7iyjl2hp1vb-python-2.7.14-env.drv'...
collision between `/nix/store/3d8nq34016rjz5vfy7w36k77nnrvkchk-gst-python-1.10.4/lib/python2.7/site-packages/gi/overrides/__init__.pyo' and `/nix/store/xir2hwxj8870j311hx8jdi63vcspljs0-python2.7-pygobject-3.24.1/lib/python2.7/site-packages/gi/overrides/__init__.pyo'
builder for '/nix/store/z32xpl22v64vhw91pcd3d7iyjl2hp1vb-python-2.7.14-env.drv' failed with exit code 25
cannot build derivation '/nix/store/lxmzpva4spmz3l4ny79r3lqcwirjsk1x-unit-mopidy-scan.service.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/z583bmd414jflbd33d440zgpkzpg44d2-unit-mopidy.service.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/hbyfiv3x1hq4gm7jdcsd77fhjwjmzjjf-system-units.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/p3jf9jjf8ws884cs20jk2h6ki2yg6zks-etc.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/2x07d2aqs981hvkd148wmssylw6wpqxm-nixos-system-dn-17.09.3109.48a76ca6f57.drv': 1 dependencies couldn't be built
error: build of '/nix/store/2x07d2aqs981hvkd148wmssylw6wpqxm-nixos-system-dn-17.09.3109.48a76ca6f57.drv' failed
```

### Steps to reproduce

AFAIK this triggers it:
```
{
  services.mopidy.enable = true;
}
```
But I can't rule out the possibility that something else is interfering with mopidy's build (is that even possible?).

## Technical details

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.

 - system: `"x86_64-linux"`
 - host os: `Linux 4.9.82, NixOS, 17.09.3047.8bce347f02f (Hummingbird)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 1.12pre5873_b76e282d`
 - channels(root): `"nixos-17.09.3109.48a76ca6f57"`
 - channels(delta): `"nixpkgs-18.03pre129212.098c7f3d945"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`

**Comment by rnhmjoj** [comment]
It happens whenever gst-python and pygobject3 are exposed together. I could try backport it but I have to find how it was fixed in master first.

**Comment by mattfredwill** [comment]
I've gotten this working temporarily by creating a local copy of the mopidy service module from master, still using 17.09 packages, tested local files and spotify.

So it seems all that's necessary is 85b84527f636c60bd8c0f0567bb471d491fb5a89

**Comment by yegortimoshenko** [comment]
Closed via 2070830ba845b69c7c597537786d9b9498fa4b28.
